applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -195,10 +230,35 @@ static const char *const channel_type_str[] = {
 	[CLOUD_CHANNEL_DEVICE_INFO] = CLOUD_CHANNEL_STR_DEVICE_INFO,
 	[CLOUD_CHANNEL_LIGHT_SENSOR] = CLOUD_CHANNEL_STR_LIGHT_SENSOR,
 };
-
+struct cloud_sensor_ch_cfg sensor_cfg[] = {
+	{ .ch = CLOUD_CHANNEL_TEMP,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .ch = CLOUD_CHANNEL_HUMID,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .ch = CLOUD_CHANNEL_AIR_PRESS,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .ch = CLOUD_CHANNEL_AIR_QUAL,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .ch = CLOUD_CHANNEL_LIGHT_RED,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .ch = CLOUD_CHANNEL_LIGHT_GREEN,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .ch = CLOUD_CHANNEL_LIGHT_BLUE,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .ch = CLOUD_CHANNEL_LIGHT_IR,
+	  .cfg = { .value = { [SENSOR_CH_CFG_ITEM_TYPE_SEND_ENABLE] = true } } }
+};
 static cloud_cmd_cb_t cloud_command_cb;
 struct cloud_command cmd_parsed;
 
+static int
+cloud_cmd_handle_sensor_set_ch_cfg(struct cloud_command const *const cmd);
```

Comment:
I think convention is to instead break the prototype up at the parameters if it exceed 80 chars, but personally no strong feelings about it and can't find anything in the code style about it

Reply:
i believe this was clang-format's doing 🙃